### ISSUE TITLE
Decide how to handle swear word list

### ISSUE THREAD

**Issue by gmemstr** [gmemstr/no-swears#6]
Currently all the words are committed in the repository as [swearwords.txt](https://github.com/gmemstr/no-swears/blob/master/swearwords.txt), I want to find a better solution for removing the need to have this committed - either downloading on the fly wordlists from somewhere (which could allow for updating) or something else.

Just want to get the conversation going and see what people think is the best solution.

**Comment by wenzelmkay** [comment]
So as of right now, if someone wants an updated swearlist, they have to update the NPM package? Are you wanting them to be able to have access to an updated list without having to update the NPM package?

I guess my question would be what is the use case for the no-swears package. I have some ideas but would like more info so I can help you brainstorm!

**Comment by gmemstr** [comment]
Yeah, currently the only way to update it would be updating the npm package. I'd rather have it behave somewhat like an adblocker where it can update the list on it's own - plus we'd also have an open wordlist for anyone else to use.

**Comment by InSertCod3** [comment]
You can host on a external server but instead of using a plain (.txt) file you can use a json structure and make GET requests like "https://someserver.com?q=swearword"

which can return like:
{
   "status": "ok",
    "is_swear": "false",
    "word": "swearword"
}
